src/entries/popup/components/CommandK/pageConfig.ts

```diff
@@ -3,69 +3,69 @@ import { i18n } from '~/core/languages';
 import { SearchItem, SearchItemType } from './SearchItems';
 
 interface Page {
-  listTitle: string | ((command: SearchItem) => string);
-  searchPlaceholder: string;
-  emptyLabel?: string;
+  listTitle: (command: SearchItem | null) => string;
+  searchPlaceholder: () => string;
+  emptyLabel?: () => string;
 }
 
 export const PAGES: { [KEY: string]: Page } = {
   HOME: {
-    listTitle: i18n.t('command_k.pages.home.section_title'),
-    searchPlaceholder: i18n.t('command_k.pages.home.search_placeholder'),
+    listTitle: () => i18n.t('command_k.pages.home.section_title'),
```

Comment:
Same rule applies here. If same object reference is used outside the component and re-render occurs we'll not get the latest `i18n.t(...)` string value. We'll use function as a method here to forcefully call it during the component re-render, so we can get the latest i18n string.